src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -180,6 +179,22 @@ protected function getArgument(\ReflectionParameter $parameter, ?string $type)
         return parent::getArgument($parameter, $type);
     }
 
+    protected static function register(parent $runtime): parent
+    {
+        $self = new self($runtime->options);
+        $runtime->options['runtimes'] += [
```

Comment:
This will fail because `$runtime->options['runtimes']` is not defined.

Reply:
fixed, thanks!